pom.xml

```diff
@@ -184,7 +184,7 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.22.2</version>
                     <configuration>
-                        <argLine>-Xmx130m -Xms130m</argLine>
+                        <argLine>-Xmx130m -Xms130m -Duser.language=en</argLine>
```

Comment:
Sorry I did not follow the language discussion fully, is this related here?

Reply:
Sorry, forgot to clean history here before submitting. Will rebase the patch.

Reply:
Ok but if you are rebasing this we can just as well just merge #2063 :) ?

Reply:
Right, since it is included